src/kfactory/kcell.py

```diff
@@ -4581,12 +4631,576 @@ def _check_cell_ports(p1: Port, p2: Port) -> int:
     return check_int
 
 
+class VKCell(BaseModel, arbitrary_types_allowed=True):
```

Comment:
Why not inherit from KCell?

Reply:
Because it is not attached to a KCell, nor do we really want everything from a KCell. We only want a slim subset and more importantly, it needs to not be attached to a layout object